src/content/getting-started/create-workload-cluster/index.md

```diff
@@ -48,9 +48,19 @@ kubectl config use-context gs-wombat
 
 ## Step 2: Template and create the workload cluster
 
-You will now create resources with `kubectl gs`. In particular, this tutorial uses the `kubectl gs template` command to create valid YAML for each resource. Alternatively, the Web UI provides a visual way to create clusters.
+First, please choose a name for the new workload cluster. We recommend you choose a naming scheme suiting your organization, and then stick to it.
 
-First, template a cluster ([command reference]({{< relref "/use-the-api/kubectl-gs/template-cluster" >}})):
+- The maximum cluster name length is 20 characters.
+- Letters, digits and dashes are allowed. Dashes are not possible at beginning or end.
+- For example, `dev-eu-central-1-a`, `dev-a`, `prod-b`, `ecommerce-dev-eu`, `prod-us` or `staging01` are all valid names.
+- The cluster name will be part of domain names such as `api.<cluster name>.<base domain>` (for instance: `api.mycluster.eu-west-1.aws.gigantic.io` or `ingress.mycluster.mycompany.com`). Since DNS records can normally be resolved publically by anyone, you should avoid encoding sensitive information into the cluster name (`mycluster` in our examples).
+- Clusters can't be renamed after creation.
+- Within one management cluster, all workload cluster names must be unique, even across namespaces. This is so that tooling and operators can look up workload clusters by their unique name without the chance of confusion.
```

Comment:
Do we require different management clusters to use different base domains? Unless we do, clusters can't use the same name even on different management clusters, or DNS names will collide.

Reply:
You're right. I extended the description more and gave a strong, general recommendation for uniqueness with more explanation.